test/unit/app/models/test_models.py

```diff
@@ -207,10 +207,37 @@ def test_approve_vendor_transfer_account(new_transfer_account):
     THEN check a VENDOR is NOT disbursed initial balance
     """
     new_transfer_account.is_vendor = True
-    new_transfer_account.approve()
+    new_transfer_account.approve_and_disburse()
 
     assert new_transfer_account.balance == 0
 
```

Comment:
We need to split out `test_models` to map the split out models folder

Reply:
yep agreed.